README.md

```diff
@@ -33,6 +33,7 @@ _These assume you have `egghead-systems` running locally_
 - **Package management:** Yarn
 - **Scripts:** Yarn
 - **JS Flavor:** ES2016+ _(default `create-react-app`)_
+- **JS Utils:** Lodash
```

Comment:
In the past I've mostly done vanilla, but I'm going full-on lodash for you Joel ;) Well, and because I see the benefits of the abstraction.